### PR TITLE
Add PCI-DSS Rules; aide,rpm,hids,dconf

### PR DESCRIPTION
**Pull request by anivan-suse**
#### Description:

- Rules Updated here:
    cce@sle15: CCE-85787-0 aide_build_database
    cce@sle15: CCE-85788-8 rpm_verify_hashes
    cce@sle15: CCE-85789-6 install_hids
    cce@sle15: CCE-85783-9 dconf_gnome_screensaver_idle_activation_enabled

### CODE REVIEW

File: linux_os/guide/system/software/integrity/software-integrity/aide/aide_build_database/ansible/shared.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_all
+# platform = multi_platform_all,multi_platform_sle
```
Comment (jan-cerny) [inline linux_os/guide/system/software/integrity/software-integrity/aide/aide_build_database/ansible/shared.yml:1]: The `multi_platform_all` includes `multi_platform_sle` which means that there is no need to add this.

Comment (anivan-suse) [reply linux_os/guide/system/software/integrity/software-integrity/aide/aide_build_database/ansible/shared.yml]: Ah yes, copy paste error. Fixed.

File: linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/rule.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel7,rhel8,rhel9,rhv4,wrlinux1019
+prodtype: fedora,ol7,ol8,rhel7,rhel8,rhel9,rhv4,wrlinux1019,sle15
```
Comment (vojtapolasek) [inline linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/rule.yml:3]:
```suggestion
prodtype: fedora,ol7,ol8,rhel7,rhel8,rhel9,rhv4,sle15,wrlinux1019
```

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Please rebase, there is one cosmetic but not unimportant suggestion.
othervise looks good.

File: linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/rule.yml (hunk 2 above)
Comment (anivan-suse) [reply linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/rule.yml]: Fixed! Thanks for the catch :)

**Comment by ggbecker** [comment]
@anivan-suse unfortunately after merging #7325, some conflicts were created. Can you solve them so we can merge this one as well? Thank you

File: products/sle15/profiles/pci-dss.profile [hunk 3]
```diff
@@ -68,11 +72,17 @@ selections:
...
     - audit_rules_usergroup_modification_shadow
     - audit_rules_sysadmin_actions
+    - dconf_gnome_screensaver_idle_activation_enabled
     - display_login_attempts
     - file_groupowner_grub2_cfg
     - file_owner_grub2_cfg
     - gid_passwd_group_same
+    - install_hids
     - no_empty_passwords
+<<<<<<< HEAD
```
Comment (ggbecker) [inline products/sle15/profiles/pci-dss.profile:82]: there are some conflicts here. I think it should be easier if you start from scratch on this one.

**Pull request closed without merging**